### DISCUSSION TITLE
Archiving index

### DISCUSSION THREAD

**Discussion by Purple-Tentacle** [ericaltendorf/plotman#432] [Q&A]
I want to archive to different drives from each plotter.
Eg 6 plotters, three of them archive to drive A, three to drive B.
Is this what the index is for?
How do I define the index for a drive?

**Comment by altendky** [comment]
Yes, that is what it is for.  Mostly it is about indicating to plotman to use different disks, not indicating exactly what disk to use.  If you have two drives then you could set half the plotters to `index: 0` and the other half to `index: 1`.  But, do you really have more plotters than final storage disks?

**Reply by Purple-Tentacle** [reply]
Okay, I think I inderstood, but have another question. 

Index 0 is always the first drive? What happens if the first drive is full? Does the plotter archive to another disk? Which one?

Currently I have 6 plotters and 2 disks, more are coming in today.

**Reply by Purple-Tentacle** [reply]
I think I can answer my own question with yes, index 0 choses the first deive with enough free space. 

> archdir = ''
    available = [(d, space) for (d, space) in archdir_freebytes.items() if 
                 space > 1.2 * plot_util.get_k32_plotsize()]
    if len(available) > 0:
        index = min(dir_cfg.archive.index, len(available) - 1)
        (archdir, freespace) = sorted(available)[index]

**Reply by mharsch** [reply]
Did we lose index functionality in 0.4? It seems to have disappeared from the documentation.

**Reply by altendky** [reply]
Just the latter, sorry.  It goes at `archiving:` `index:`.
